make/config/traefik/traefik-values.yaml

```diff
@@ -0,0 +1,28 @@
+dditionalArguments:
```

Comment:
That table is really well presented :+1: 

> Since we don't use Traefik in the e2e, should I remove e2e-setup-traefik and e2e-setup-haproxyingress altogether?

We could maybe add like a `$(warning These targets are provided for information only and may be removed)`? They're probably still useful for people but if we're not actively using them they could rot or break over time. I don't really feel strongly though, I'd happily just leave them or remove them.

Reply:
I went with the warning idea in 3101b8268.